docs/configuration.v2alpha1.md

```diff
@@ -40,9 +40,13 @@ spec:
 | features.admissionController.mutateUnlabelled | MutateUnlabelled enables config injection without the need of pod label 'admission.datadoghq.com/enabled="true"'. Default: false |
 | features.admissionController.serviceName | ServiceName corresponds to the webhook service name. |
 | features.admissionController.webhookName | WebhookName is a custom name for the MutatingWebhookConfiguration. Default: "datadog-webhook" |
-| features.apm.enabled | Enabled enables Application Performance Monitoring. Default: false |
+| features.apm.enabled | Enabled enables Application Performance Monitoring. Default: true |
 | features.apm.hostPortConfig.enabled | Enabled enables host port configuration Default: false |
 | features.apm.hostPortConfig.hostPort | Port takes a port number (0 < x < 65536) to expose on the host. (Most containers do not need this.) If HostNetwork is enabled, this value must match the ContainerPort. |
+| features.apm.singleStepInstrumentation.disabledNamespaces | DisabledNamespaces specifies the namespaces of the pods to not inject the Datadog APM libraries into. |
```

Comment:
Can we please change the configuration names to:
`features.apm.instrumentation.enabled`
`features.apm.instrumentation.enabledNamespaces`
`features.apm.instrumentation.disabledNamespaces`
`features.apm.instrumentation.libVersions`

These names will align with the naming used in Helm Charts.